include/CXXGraph/Node/Node_impl.hpp

```diff
@@ -65,6 +65,11 @@ const T &Node<T>::getData() const {
   return data;
 }
 
+template <typename T>
+T &Node<T>::getData() {
```

Comment:
Same here - the `&` should be next to the `T`.

Reply:
will update